src/pybind/mgr/pg_autoscaler/module.py

```diff
@@ -670,6 +670,11 @@ def _get_pool_status(
         ret, _, _ = self._get_pool_pg_targets(osdmap, even_pools, crush_map, root_map,
                                          pool_stats, ret, threshold, 'third', overlapped_roots)
 
+        # If noautoscale flag is set, we set pg_autoscale_mode to off
+        if self.has_noautoscale_flag():
+            for p in ret:
+                p['pg_autoscale_mode'] = 'off'
```

Comment:
I'm a confused as to why this commit is introducing has_noautoscale_flag() while removing its only usage.